core/embed/bootloader/main.c

```diff
@@ -29,7 +29,12 @@
 #include "mpu.h"
 #include "random_delays.h"
 #include "secbool.h"
+#ifdef TREZOR_MODEL_T
 #include "touch.h"
+#endif
+#if defined TREZOR_MODEL_R || defined TREZOR_MODEL_1
```

Comment:
fixed 572cc5b7a81e0a46b8db143ba423b7caa8449536 and 2e8dd0fcdb26cf90ddcaffbd48665ac75a056366. Though we still have model 1 variant in bootloader SConscript. 

However, this breaks the compilation of bootloader for model 1 variant. Since bootloader is part of build_all and build_embed, this also breaks. Which makes me wanna drop the commits. Or i can change these particular conditionals to not throw "Unknown Trezor model" errors.

Reply:
> can change these particular conditionals to not throw "Unknown Trezor model" errors.

that seems like the best option

Reply:
but I'm wondering if we could somehow noop-ize bootloader build for T1. Like, `Sconscript.bootloader` would just print "not building for T1" instead of doing anything

Reply:
This can be achieved with this placed at the beginning of sconscript, seems like a better choice then running dummy compilation:
`if TREZOR_MODEL in ('1', ):
    # skip bootloader build
    env = Environment()
    def build_bootloader(target,source,env):
        print(f'Bootloader: nothing to build for Model {TREZOR_MODEL}')
    program_bin = env.Command(
                 target='bootloader.bin',
                 source=None,
                 action=build_bootloader
             )
    Return()
`
We can include this also in boardloader and bootloader_ci to skip those too.

Reply:
fixed 2ba1b65bf249d6a80497222e9670406d8b3f5acc